src/components/FilterCollapse.js

```diff
@@ -48,10 +48,16 @@ const numFields = [
 const colorFields = ['color', 'identity'];
 
 const FilterCollapse = ({ isOpen, hideDescription }) => {
+  const { filterInput, setFilterInput, filterValid, filterResult } = useContext(CubeContext);
+
   const [advancedOpen, setAdvancedOpen] = useState(false);
   const [values, setValues] = useState({});
+  const [searchFilterInput, setSearchFilterInput] = useState(filterInput);
 
-  const { filterInput, setFilterInput, filterValid, filterResult } = useContext(CubeContext);
+  const applySearchFilter = useCallback(
```

Comment:
I don't think this needs to be a callback. It's a single-line function, so the cost of creating it is negligible, and it's only passed to a single button, so we're not really saving too much on child rerenders either.

Reply:
Agreed. That is also what I thought but looking around I noticed that all functions seemed to use it so I just followed the pattern. I will remove it.